src/modules/site-v2/templates/tools/heritability_calculator/result.html

```diff
@@ -52,9 +52,16 @@
       {% endif %}
       {% endif %}
     </p>
-    {%- if result %}
+    {%- if result and result['broad-sense'] %}
```

Comment:
do we need a default value in `broad-sense` is not set?

Reply:
Might be worth asking Erik? This should only be possible if there was an error running the tool (in which case the whole report would be an error) or if something went wrong in the storage / retrieval...